src/components/Sum.jsx

```diff
@@ -1 +1,19 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {a + b}
+    </p>
```

Comment:
Don't forget to check yourself with the checklist
![image](https://user-images.githubusercontent.com/96409563/207805252-96d2c71f-30c4-40c3-8d53-95831f97d082.png)